core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -366,6 +370,7 @@ var TopicsAndSkillsDashboardPage = function() {
       unusedSkillsTabButton,
       'Unused skills tab button taking too long to be clickable');
     await unusedSkillsTabButton.click();
+    await this.filterSkillsByStatus('Unassigned');
```

Comment:
Now that this function also does this filtering, can you change the function name to reflect the filtering?

Reply:
Seperated these.